packages/loader/container-loader/src/container.ts

```diff
@@ -1119,7 +1119,11 @@ export class Container
 		);
 		assert(!!this.baseSnapshot, 0x5d4 /* no base snapshot */);
 		assert(!!this.baseSnapshotBlobs, 0x5d5 /* no snapshot blobs */);
-		const pendingRuntimeState = await this.runtime.getPendingLocalState(props);
+		const pendingRuntimeState = await PerformanceEvent.timedExecAsync(
```

Comment:
it might be better to wrap this whole method, and then add a similar wrapping in the runtime, rather than grabbing only part here

Reply:
You mean wrap getPendingLocalStateCore calls? and move this timedExecAsync inside?

Reply:
You mean wrap getPendingLocalStateCore calls? and move this timedExecAsync inside?

Reply:
wrap everything that happens in this method, it will track the pref at the container level, and then do the same at the runtime level, and maybe even blob manager, so each layer tracks its totally perf, rather than the top layer trying to sub-divide it.